project.clj

```diff
@@ -104,19 +103,21 @@
                                      [org.clojure/clojure "1.11.0-master-SNAPSHOT" :classifier "sources"]]}
 
              :test {:dependencies [[org.clojure/java.classpath "1.0.0"]]
-                    :resource-paths ["test-resources"
-                                     "not-a.jar"
-                                     "does-not-exist.jar"]
+                    :resource-paths ~(cond-> ["test-resources"
+                                              "not-a.jar"
+                                              "does-not-exist.jar"]
+                                       jdk8? (conj (unzipped-jdk-source))
```

Comment:
"./unzipped-jdk-source/"  should be something like `~/.cider/orchard/<hash of jdk>/unzipped-jdk-source`

Reply:
Won't <version-of-the-jdk> be a bit more readable than a hash? I

Reply:
> Won't be a bit more readable than a hash?

Finally I'm doing `(->> "java.version" System/getProperty (re-seq #"\d+") (string/join))` as of my wip